AudibleBoulders/AudibleBoulders
# Setup page

#### What's this PR do?
Setup page serves gitSpy folder in a zipfile.
Populate setup page with unique setup instructions

#### What are the important parts of the code?
src/client/app/setup/setup.html
src/client/app/setup/setup.js

#### How should this be tested by the reviewer?
If you are testing on local, you need to change line 67 in gitSpy/gitSpy.js in the repo you are testing from gitspy to localhost:8080 
You also need to change line 49 in server.js in our code  from gitspy to localhost:8080

Add a new dashboard!
Follow setup instructions
Make a commit and see if it populates the database, woot!

#### Is any other information necessary to understand this?
Ask me question if anything is confusing

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

#### Screenshots (if appropriate)
